pkg/model/api/config/config_test.go

```diff
@@ -352,6 +362,9 @@ frontend:
       dashboard: "DuplicatePanels"
     - project: "Unknown"
       dashboard: "Dashboard"
+
+plugins:
+  path: "custom/plugins"
```

Comment:
the tests are not passing, because you put the plugins config in the middle of the frontend config :)

Reply:
silly me, thnx for the hint. It should be ready now.